components/hab/src/command/plan.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2016 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod initialize {
+    use std::fs::create_dir_all;
+    use std::fs::File;
+    use std::io::Write;
+    use std::path::Path;
+
+    use common::ui::{UI, Status};
+    use error::Result;
+
+    const PLAN_TEMPLATE: &'static [u8] = b"test plan.sh";
+    const RUN_HOOK_TEMPLATE: &'static [u8] = b"test run hook";
+
+    pub fn start(ui: &mut UI) -> Result<()> {
+        try!(ui.begin("Constructing a cozy habitat for your app..."));
+        try!(ui.br());
+
+        try!(create_with_template(ui, "habitat/plan.sh", PLAN_TEMPLATE));
```

Comment:
As far as I can tell the canonical location is `WORKING_DIR/habitat/plan.sh` according to https://www.habitat.sh/tutorials/getting-started/linux/create-plan/

> The /habitat directory contains the heart of a Habitat package: the plan. This is a directory comprised of shell scripts and optional configuration files that define how you download, configure, make, install, and manage the lifecycle of the software in the package. Currently, the plan in /habitat-example-plans/habitat only contains plan.sh, which you will populate and add to in the following sections.